lib/WebSocket.js

```diff
@@ -188,17 +188,13 @@ class WebSocket extends EventEmitter {
     this.emit('close', this._closeCode || 1000, this._closeMessage || '');
 
     if (this._socket) {
-      if (this._ultron) this._ultron.destroy();
```

Comment:
If `this._socket` is truthy then `this._ultron` is also truthy as they are cleared together.